HLTrigger/Configuration/python/HLT_75e33/sequences/HLTHgcalTiclPFClusteringForEgammaUnseededSequence_cfi.py

```diff
@@ -14,5 +14,17 @@
 from ..modules.ticlLayerTileProducer_cfi import *
 from ..modules.ticlSeedingGlobal_cfi import *
 from ..modules.ticlTrackstersCLUE3DHigh_cfi import *
+from ..modules.hltHgcalSoARecHitsProducer_cfi import *
+from ..modules.hltHgcalSoARecHitsLayerClustersProducer_cfi import *
+from ..modules.hltHgcalSoALayerClustersProducer_cfi import *
+from ..modules.hltHgcalLayerClustersFromSoAProducer_cfi import *
 
 HLTHgcalTiclPFClusteringForEgammaUnseededSequence = cms.Sequence(hgcalDigis+HGCalUncalibRecHit+HGCalRecHit+particleFlowRecHitHGC+hgcalLayerClustersEE+hgcalLayerClustersHSci+hgcalLayerClustersHSi+hgcalMergeLayerClusters+filteredLayerClustersCLUE3DHigh+ticlSeedingGlobal+ticlLayerTileProducer+ticlTrackstersCLUE3DHigh+particleFlowClusterHGCalFromTICLUnseeded+particleFlowSuperClusterHGCalFromTICLUnseeded)
+
+_HLTHgcalTiclPFClusteringForEgammaUnseededSequence_heterogeneous = cms.Sequence(hgcalDigis+HGCalUncalibRecHit+HGCalRecHit+particleFlowRecHitHGC+hltHgcalSoARecHitsProducer+hltHgcalSoARecHitsLayerClustersProducer+hltHgcalSoALayerClustersProducer+hltHgCalLayerClustersFromSoAProducer+hgcalLayerClustersHSci+hgcalLayerClustersHSi+hgcalMergeLayerClusters+filteredLayerClustersCLUE3DHigh+ticlSeedingGlobal+ticlLayerTileProducer+ticlTrackstersCLUE3DHigh+particleFlowClusterHGCalFromTICLUnseeded+particleFlowSuperClusterHGCalFromTICLUnseeded)
+
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
```

Comment:
For internal TSG consideration, this type of pattern was already introduced at https://github.com/cms-sw/cmssw/pull/45178, but it is somehow at odds with the current (Run3) customization paradigm. Should we consider creating a `customizeHLTForCMSSW`-equivalent for the phase2 menu and put there all the necessary customizations for alpaka? 
That way it will be easier to remove when the  `alpaka` modifier becomes the "norm".
I guess this also depends very much on how we want to store the phase2 menu in the long term. 
 @VourMa @SohamBhattacharya @fwyzard @Martin-Grunewald @missirol